src/Microsoft.AspNet.Mvc.ViewFeatures/Logging/ValidateAntiforgeryTokenAuthorizationFilterLoggerExtensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNet.Mvc.Logging
+{
+    public static class ValidateAntiforgeryTokenAuthorizationFilterLoggerExtensions
+    {
+        private static readonly Action<ILogger, string, Exception> _antiforgeryTokenInvalid;
+
+        static ValidateAntiforgeryTokenAuthorizationFilterLoggerExtensions()
+        {
+            _antiforgeryTokenInvalid = LoggerMessage.Define<string>(
+                LogLevel.Error,
+                1,
+                "Antiforgery token validation failed. {Message}");
+        }
+
+        public static void AntiforgeryTokenInvalid(this ILogger logger, string message, Exception exception)
+        {
+            if (logger.IsEnabled(LogLevel.Error))
```

Comment:
This check isn't needed unless you're doing something complex/expensive in here. Just skip it.